### ISSUE TITLE
reminder: fix AJV findings

### ISSUE THREAD

**Issue by jkowalleck** [CycloneDX/specification#208]
I just tried a new JSON schema validator, [`AJV`](https://ajv.js.org/), and this tool found some issues in current schema 1.4, that should be fixed in 1.5

Need to dig into the details. here are quick findings.
```text
strict mode: missing type "object" for keyword "additionalProperties" at "http://cyclonedx.org/schema/bom-1.4.schema.json#/properties/affects/items/properties/versions/items" (strictTypes)
strict mode: missing type "object" for keyword "properties" at "http://cyclonedx.org/schema/bom-1.4.schema.json#/properties/affects/items/properties/versions/items" (strictTypes)
strict mode: missing type "object" for keyword "required" at "http://cyclonedx.org/schema/bom-1.4.schema.json#/properties/affects/items/properties/versions/items/oneOf/0" (strictTypes)
strict mode: required property "version" is not defined at "http://cyclonedx.org/schema/bom-1.4.schema.json#/properties/affects/items/properties/versions/items/oneOf/0" (strictRequired)
strict mode: missing type "object" for keyword "required" at "http://cyclonedx.org/schema/bom-1.4.schema.json#/properties/affects/items/properties/versions/items/oneOf/1" (strictTypes)
strict mode: required property "range" is not defined at "http://cyclonedx.org/schema/bom-1.4.schema.json#/properties/affects/items/properties/versions/items/oneOf/1" (strictRequired)
strict mode: "additionalItems" is ignored when "items" is not an array of schemas
```

!! another thing: 
- remove the `bom.version` from the required properties in JSON spec.
  Having it required just makes no sense, as the property has a default value. 
  a default value is to be used when the element was missing – why would it be required, if it was allowed to be missing in the beginning??? see #216

**Comment by jkowalleck** [comment]
will research the reasons and try to fix them in current 1.5 draft

**Comment by stevespringett** [comment]
Perhaps wait until more things are merged into the 1.5 branch so we can do them all at once?

**Comment by jkowalleck** [comment]
fixed by https://github.com/CycloneDX/specification/pull/230
